ansible/inventory/group_vars/operator-pipeline.yml

```diff
@@ -1,6 +1,6 @@
 ##### General settings
 env: unset
-oc_namespace: operator-pipeline-{{ env }}
+oc_namespace: "{{ (namespace) if (namespace) is defined else operator-pipeline-(env) }}"
```

Comment:
If you pass the oc_namespace/namespace as env variable when running a playbook the value from inventory gets overwritten and you don't have to do this.

Reply:
Good point! Applied.